### PR TITLE
[PLAY-970] Website - Changelog updates

### PR DESCRIPTION
**Pull request by nickamantia**
[PLAY-970](https://nitro.powerhrg.com/runway/backlog_items/PLAY-970)

This PR updates styling for the `Changelog.md` file

New Styles:
Screenshot 2023-09-13 at 10 25 26 AM

Blockquote Example (this is not actually in blockcode, just added it for an example image):
Screenshot 2023-09-13 at 10 33 33 AM

**How to test?** Steps to confirm the desired behavior:
1. Go to Changelog page and confirm that new styles are correct.

To Test blockquotes:
1. Pull down branch locally
2. Make something in `Changelog.md` blockquote by adding a `> `  at the start if the line.
3. check local env to see if styles look correct.

#### Checklist:
- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

**Review by jasperfurniss — APPROVED** [review]

**Review by nidaqg — DISMISSED** [review]

File: playbook-website/app/javascript/site_styles/docs/_markdown.scss [hunk 1]
```diff
@@ -59,6 +59,11 @@
       transition: 0.1s all ease-in-out;
     }
   }
+  h4:last-of-type {
+    border-top: 1px solid $border_light;
+    padding-top: $space_md;
+    font-weight: $regular;
+  }
```
Comment (nidaqg) [inline playbook-website/app/javascript/site_styles/docs/_markdown.scss:66]: Everything looks great @nickamantia , but I think you'll need to nest this inside the changelog classname as well. It's effecting the markdown styling for h4 that we have for the website
Here's what the guides/design_guidelines/example_markdown_styling page looks like on prod:
Screenshot 2023-09-14 at 8 48 22 AM

And this is on your review env:
Screenshot 2023-09-14 at 8 48 29 AM

cc: @jasperfurniss

Comment (nickamantia) [reply playbook-website/app/javascript/site_styles/docs/_markdown.scss:66]: great catch! gonna fix this right now! thanks!!

**Review by nidaqg — DISMISSED** [review]

File: playbook-website/app/javascript/site_styles/docs/_markdown.scss [hunk 2]
```diff
@@ -123,6 +123,33 @@
...
+    left: 0;
+    top: 0;
+  }
+
+  .changelog-container h4:last-of-type {
+    border-top: 1px solid $border_light;
+    padding-top: $space_md;
+    font-weight: $regular;
+  }
+
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/docs/_markdown.scss:152]: Two suggestions:

1. Why not move this code into it's own file. This file controls ALL the markdown for the entire site. This override mixed into the middle makes understanding this file more difficult. Overrides should be separated to improve clarity.
2. Utilize SASS power. You are repeating selectors when nesting will save you time and keep this DRY.

```suggestion
.changelog-container {
  blockquote {
    position: relative;
    font-style: normal;
    padding: 0px 0px 0px calc(#{$space_md} - 1px);
    &::before {
      content: '';
      display: inline-block;
      background: $border_light;
      width: 7px;
      height: 100%;
      border-radius: 5px;
      position: absolute;
      left: 0;
      top: 0;
    }
    li, p {
      font-size: $text_medium;
    }
  }
  h4:last-of-type {
    border-top: 1px solid $border_light;
    padding-top: $space_md;
    font-weight: $regular;
  }
}
```
Again I suggest moving it from the middle.

Comment (nickamantia) [reply playbook-website/app/javascript/site_styles/docs/_markdown.scss]: moved styles to a separate file and nested selectors. thanks!

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**